📚 Docs Refresh: Files Source (#6663)
airbytehq/airbyte: docs/integrations/sources/file.md

| Parquet | Yes |
| Pickle | No |

## Getting Started (Airbyte Cloud)

> Setup through Airbyte Cloud will be exactly the same as the open-source setup, except for the fact that local files are naturally disabled.

Reviewer: ```suggestion
Setup through Airbyte Cloud will be exactly the same as the open-source setup, except for the fact that local files are disabled.
```

Please see \(or add\) more at `airbyte-integrations/connectors/source-file/integration_tests/integration_source_test.py` for further usages examples.

## Performance Considerations and Notes

> In order to read large files from a remote location, we are leveraging the capabilities of [smart\_open](https://pypi.org/project/smart-open/). However, it is possible to switch to either [GCSFS](https://gcsfs.readthedocs.io/en/latest/) or [S3FS](https://s3fs.readthedocs.io/en/latest/) implementations as it is natively supported by the `pandas` library. This choice is made possible through the optional `reader_impl` parameter.

Reviewer: ```suggestion
In order to read large files from a remote location, this connector uses the [smart\_open](https://pypi.org/project/smart-open/) library. However, it is possible to switch to either [GCSFS](https://gcsfs.readthedocs.io/en/latest/) or [S3FS](https://s3fs.readthedocs.io/en/latest/) implementations as it is natively supported by the `pandas` library. This choice is made possible through the optional `reader_impl` parameter.
```
